examples/apps/collaborative-textarea/package.json

```diff
@@ -38,16 +38,16 @@
 		"webpack:dev": "webpack --env development"
 	},
 	"dependencies": {
-		"@fluid-example/example-utils": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
-		"@fluid-experimental/react-inputs": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
-		"@fluidframework/aqueduct": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
+		"@fluid-example/example-utils": "^2.0.0-internal.5.0.0",
```

Comment:
I think we want to keep "<" version ranges.

Reply:
Good catch! I'm investigating to see if we have a bug in the bump tools.

Reply:
Found the bug: #14727